classes/autoptimizeConfig.php

```diff
@@ -539,4 +551,46 @@ static function ao_admin_tabs(){
         
         return $tabContent;
     }
+
+	public function load_custom_wp_admin_style()
+	{
+		wp_enqueue_style( 'autoptimize-toolbar', plugins_url( 'autoptimize/css/toolbar.css' ), array(), time(), "all" );
+	}
+    
+	public function load_custom_wp_admin_javascript()
+	{
+		wp_enqueue_script( 'autoptimize-toolbar', plugins_url( 'autoptimize/js/toolbar.js' ), array(), time(), true );
+	}
+	
+	public function load_admin_toolbar()
+	{
+		$show = false;
+
+		// Admin
+		$show = ( current_user_can( 'manage_options' ) || current_user_can( 'edit_others_pages' ) ) ? true : false;
+
+		// Author
+		if( defined( 'AUTOPTIMIZE_TOOLBAR_FOR_AUTHOR' ) && AUTOPTIMIZE_TOOLBAR_FOR_AUTHOR )
```

Comment:
Sorry, I was wrong. 
I wanted to give the chance to the admin that can enable the editor to do this, not the author.
Because usually I suggest the owners of a site not using the administrator user to make changes in publications but with the editor, and when this happens, the changes are not reflected if they are not allowed to empty the cache.
What do you think about it?